### PR TITLE
Add boost.coroutine@1.83.0.bcr.2, which includes the library's cpp sources.

### PR DESCRIPTION
**Pull request by benjaminp**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @wep21, @Vertexwahn, modules you maintain (boost.coroutine) have been updated in this PR. Please review the changes.

**Comment by benjaminp** [comment]
@bazel-io skip_check unstable_url

File: modules/boost.coroutine/1.83.0.bcr.2/overlay/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "boost.coroutine",
+    srcs = [
+        "src/detail/coroutine_context.cpp",
+        "src/exceptions.cpp",
+    ] + select({
+        "@platforms//os:linux": ["src/posix/stack_traits.cpp"],
```
Comment (Vertexwahn) [inline modules/boost.coroutine/1.83.0.bcr.2/overlay/BUILD.bazel:11]: does it make sense to set default to "src/posix/stack_traits.cpp"?

**Review by Vertexwahn — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**